src/extension/geofence/geofence-server/pom.xml

```diff
@@ -45,19 +30,30 @@
       <groupId>org.hibernate</groupId>
       <artifactId>hibernate-core</artifactId>
     </dependency>
+
     <dependency>
-      <groupId>org.hibernatespatial</groupId>
-      <artifactId>hibernate-spatial</artifactId>
+      <groupId>org.opengeo</groupId>
+      <artifactId>geodb</artifactId>
+      <version>${opengeo-geodb.version}</version>
     </dependency>
+
+    <!-- Older versions of h2 fails in drop/create tables   -->
+    <!-- This version may break netcdf/grib modules         -->
+    <!-- Please do consider using a postgis backend instead -->
     <dependency>
```

Comment:
Wondering if we shoud have a profile to enable/disable this dependency.
Use case, starting GeoServer during development with both geofence and netcdf, with this setup it would be impossible...

Reply:
I guess that during development we need to work on the `src/web/app/pom.xml`.
We may add the profiles
 - `geofence-server-pg` --> includes postgres + postgis jars
 - `geofence-server-h2` --> includes h2 + geodb jars

Only specifying the profile `geofence-server` will not include any jdbc driver (feedback needed here @aaime)

I guess that in the same way we could create 2 different assembly files, one containing H2 related libs and the other postgres+postgis.